### PR TITLE
Added mobius inversion

### PR DESCRIPTION
**Pull request by Chillee**
I'm not sure what you meant by this: https://github.com/kth-competitive-programming/kactl/pull/159#discussion_r388126994

I'm not aware of such a version - are you thinking of what he reduced the problem to here? https://codeforces.com/blog/entry/53925

Looks like this: 

image

### CODE REVIEW

**Comment by simonlindholm** [comment]
> I'm not sure what you meant by this: #159 (comment)

https://en.wikipedia.org/wiki/M%C3%B6bius_inversion_formula#Generalizations

File: content/number-theory/chapter.tex [hunk 1]
```diff
@@ -48,3 +48,14 @@ \section{Estimates}
...
+\section{Mobius Function}
+\[
+	\mu(n) = \begin{cases} 0 & n \textrm{ is not square free}\\ 1 & n \textrm{ has even number of prime factors}\\ -1 & n \textrm{ has odd number of prime factors}\\\end{cases}
+\]
+  Mobius Inversion:
+  \[ g(n) = \sum_{d|n} f(d) \Leftrightarrow f(n) = \sum_{d|n} \mu(d)g(n/d) \]
+
+  Other useful formulas/forms:
+  \[ \sum_{d | n} \mu(d) = [ n = 1] \]
+  \[ g(n) = \sum_{n|d} f(d) \Leftrightarrow f(n) = \sum_{n|d} g(d)\mu(d/n)\]
```
Comment (simonlindholm) [inline content/number-theory/chapter.tex:61]: `g(n) = \sum_{n|d} f(d)` seems to be a sum over infinitely many d?

Comment (Chillee) [reply content/number-theory/chapter.tex:61]: https://math.stackexchange.com/questions/1965342/does-the-mobius-inversion-theorem-hold-when-sum-function-is-over-multiples-inste

and

https://codeforces.com/blog/entry/59962?#comment-475400

It's possible if `f(d) = 0` for some high enough d.

**Comment by Chillee** [comment]
Added the floor formula. Also switched to `$` instead of `\[` since it took up a lot of space. I don't know if there's a better way to do it.

Current:

image

Originally:

image

File: content/number-theory/chapter.tex (hunk 1 above)
Comment (simonlindholm) [reply content/number-theory/chapter.tex]: Oh, I see. I wonder if it's worth a comment, but not obvious what would fit. "(if convergent)" maybe

File: content/number-theory/chapter.tex [hunk 2]
```diff
@@ -48,3 +48,17 @@ \section{Estimates}
...
+
+\section{Mobius Function}
+\[
+	\mu(n) = \begin{cases} 0 & n \textrm{ is not square free}\\ 1 & n \textrm{ has even number of prime factors}\\ -1 & n \textrm{ has odd number of prime factors}\\\end{cases}
+\]
+  Mobius Inversion:
+  \[ g(n) = \sum_{d|n} f(d) \Leftrightarrow f(n) = \sum_{d|n} \mu(d)g(n/d) \]
+  Other useful formulas/forms:
+
+  $ \sum_{d | n} \mu(d) = [ n = 1] $ (very useful)
```
Comment (simonlindholm) [inline content/number-theory/chapter.tex:60]: remove this comment since it line-wraps, I guess

**Pull request merged**